invoicing/pdf.py

```diff
@@ -9,10 +9,26 @@
 from apartment.elastic.documents import ApartmentDocument
 from apartment.elastic.queries import get_apartment, get_projects
 from apartment_application_service.pdf import create_pdf, PDFData
+from customer.models import Customer
 
 INVOICE_PDF_TEMPLATE_FILE_NAME = "invoice_template.pdf"
 
 
+def _get_payer_name_and_address(customer: Customer) -> str:
+    primary_profile = customer.primary_profile
+    payer_names = primary_profile.full_name
+    if secondary_profile := customer.secondary_profile:
+        payer_names = (
+            f"{primary_profile.full_name}"
+            + _("and")
```

Comment:
Finnish translation ("ja") should be included in this PR because otherwise we will never remember to add it 😄